### PR TITLE
Recorded future solution 2.4

### PR DESCRIPTION
**Pull request by RecordedFutureOskbo**
Required items, please complete
   
   Change(s):
   - Handle 404 in enrichment playbook
   - Reduced size of image
   - Added new Sandbox enrichment playbook to solution

   Reason for Change(s):
   - Bug Fixes 
   - new playbook

  Testing Completed:
   - Yes

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @manishkumar1991, can you please provide your feedback?

**Comment by manishkumar1991** [comment]
> Hi @manishkumar1991, can you please provide your feedback?

Review this PR

**Comment by manishkumar1991** [comment]
@RecordedFutureOskbo 
Kindly change the name of all arm template playbooks to azuredeploy.json ,    as per standard

**Comment by RecordedFutureOskbo** [comment]
> @RecordedFutureOskbo Kindly change the name of all arm template playbook files to azuredeploy.json , as per standard

Renaming done
- Changed references in Solution_RecordedFuture.json
- Generated new maintemplate.json and zip
- Changed references in readme.md to new name 
- Corrected spelling mistakes in readme

**Comment by v-rbajaj** [comment]
Looking into this PR. Will soon provide an update.

**Comment by RecordedFutureOskbo** [comment]
@v-rbajaj , @manishkumar1991 , @v-atulyadav any feedback on this PR? We started to prepare for next version and don't want to  end up in merge conflict with our self. As feedback, the way the solution is packed together makes everything tangled up.

**Comment by v-rbajaj** [comment]
Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate.

**Comment by RecordedFutureOskbo** [comment]
> Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate.

Solution done! We pull all our data into the ThreatIntelligenceIndicator table. I'll try to export a representative dataset from our logic apps.

File: Solutions/Recorded Future/Playbooks/readme.md [hunk 1]
```diff
@@ -1,90 +1,274 @@
...
+
+1. [Azure Monitor Logs cost calculations and options](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/cost-logs)
+
+Recorded Futures risk lists are generated at different cadences as described in this support article [Risk List Download Recommendations](https://support.recordedfuture.com/hc/en-us/articles/115010401968-Risk-List-Download-Recommendations) (Require Recorded Future Login). It is possible to adjust cadence to decrease cost of data processing.
+
+# Installation
+
+There are two options for installing playbooks and starting automated threat response:
+
+- Installing the solution from from [Content Hub](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/dontDiscardJourney~/true/id/recordedfuture1605638642586.recorded_future_sentinel_solution). (Recommended)
```
Comment (v-rbajaj) [inline Solutions/Recorded Future/Playbooks/readme.md:76]: There are two "from" please remove one, thanks

**Comment by RecordedFutureOskbo** [comment]
> Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate. Also, can you please provide sample data?

I have added small amount of test data to the Solution. If you need data for specific testcases I can provide it to you.

**Comment by v-rbajaj** [comment]
Hi @RecordedFutureOskbo, since this is a new solution, can you please package this solution with version 2.0.0?

**Comment by RecordedFutureOskbo** [comment]
> Hi @RecordedFutureOskbo, since this is a new solution, can you please package this solution with version 2.0.0?

But this is version 2.4.0 of our solution, our current version in content hub is 2.3.0
image
image

**Review by v-rbajaj — APPROVED** [review]
Looks fine

**Pull request merged**